src/WebJobs.Script/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -67,9 +67,11 @@ private static IOpenTelemetryBuilder ConfigureMetrics(this IOpenTelemetryBuilder
         {
             return builder.WithMetrics(builder =>
             {
-                builder.AddAspNetCoreInstrumentation();
-                builder.AddMeter(HostMetrics.FaasMeterName);
-                builder.AddView(HostMetrics.FaasInvokeDuration, new ExplicitBucketHistogramConfiguration
+                builder.AddAspNetCoreInstrumentation()
+                .AddRuntimeInstrumentation()
```

Comment:
Should we disable them by default and have customers explicitly enable them (via config / app setting etc.) if needed?

Reply:
I will add the config support later; there's a config proposal in the OTel community and I am waiting for it to be accepted.